packages/bsky/src/hydration/actor.ts

```diff
@@ -95,9 +96,14 @@ export class ActorHydrator {
     const res = await this.dataplane.getActors({ dids })
     return dids.reduce((acc, did, i) => {
       const actor = res.actors[i]
+      const isNoHosted =
+        actor.takenDown ||
+        ['takendown', 'suspended', 'deactivated'].includes(
+          actor.upstreamStatus ?? '',
```

Comment:
May be able to future-proof this using something like `actor.upstreamStatus && actor.upstreamStatus !== 'active'`.  Or if this is an area where it should never be set to `active`, perhaps just `!!actor.upstreamStatus`.